requirements/xpu.txt

```diff
@@ -10,10 +10,10 @@ wheel
 jinja2>=3.1.6
 datasets # for benchmark scripts
 numba == 0.60.0 # v0.61 doesn't support Python 3.9. Required for N-gram speculative decoding
---extra-index-url=https://download.pytorch.org/whl/xpu
+
 torch==2.8.0+xpu
 torchaudio
 torchvision
-pytorch-triton-xpu
---extra-index-url=https://pytorch-extension.intel.com/release-whl/stable/xpu/us/
-intel-extension-for-pytorch==2.8.10+xpu
+--extra-index-url=https://download.pytorch.org/whl/xpu
```

Comment:
why remove triton

Reply:
It's automatically installed as dependency of torch.
![Uploading image.png…]()

Reply:
checked cuda also not explicit add triton in cuda.txt. It's ok to remove.